README.md

```diff
@@ -69,8 +70,16 @@ docker compose up --build
 
 ## Production build
 
+(adjust tags as needed)
+
 ```
-docker build -f production.Dockerfile .
+# to load the built image into local docker images
+docker build --tag rails-on-docker --file production.Dockerfile . --load
```

Comment:
One thing to note is that both `--load` and `--push` options work only when the `DOCKER_BUILDKIT=1` is set

Reply:
good catch - updated the README - https://github.com/ryanwi/rails7-on-docker/blob/main/README.md#production-build